examples/triply/vertexBufferRoot.cpp

```diff
@@ -258,7 +257,8 @@ bool VertexBufferRoot::_constructFromPly(const std::string& filename)
     ++progress;
 
     data.calculateNormals();
-    data.scale(2.0f);
+    if (_rescale)
+        data.scale(2.0f);
```

Comment:
Did you figure out what's the purpose of this scaling? It looks completely arbitrary

Reply:
Yes, it is not exactly a scaling but a normalisation to the given size. This works together with the default camera position at z=-2.0 to have a good default view of the model.